.storybook/main.ts

```diff
@@ -10,6 +10,7 @@ module.exports = {
     "@storybook/addon-interactions",
     "@storybook/addon-a11y",
     "@storybook/addon-jest",
+    "storybook-xstate-addon/preset",
```

Comment:
Added back this plugin for easier inspection of the xstate machines in Storybook